src/components/DetailsBlock.js

```diff
@@ -28,8 +32,11 @@ const DetailsBlock = (props) => {
         {/* {
             dataFormatCheck(content)? content.map()
         } */}
-        {props.children}
+        <DetailsTitle title={title} />
+        {/* If the detailRows prop is populated, map over an array of objects. 
+        Otherwise, pass children as-is. */}
+        {detailRows ? detailRows.map(row => <DetailsContent type={row.type} content={row.content} />) : children}
```

Comment:
Instead of passing <DetailsTitle> and <DetailsContent> as children in each instance of a DetailsBlock component on IncidentDetails.js, they seem to happen often enough that they could be encapsulated inside the DetailsBlock component altogether.  This cleans up your import statements as well.

The ternary operation will still allow for passing children, but the other mode is more opinionated by using DeatilsContent to handle key/value pairs of information.